### PR TITLE
[gps] nmea: send ABI msg on every valid NMEA sentence

### PR DESCRIPTION
**Pull request by flixr**
send GPS ABI message even if position is not available but a new valid sentence was received,
e.g. while no fix but get tow and satellite info

should be a better fix for #1618

same problem is probably present in some other gps drivers...

### CODE REVIEW

**Comment by flixr** [comment]
Similar problem should be fixed for sirf with b973dd2110537585942d7ccc23d952ea45c76e46

File: sw/airborne/subsystems/gps/gps_furuno.c [hunk 1]
```diff
@@ -125,4 +125,7 @@ void nmea_parse_perdcrv(void)
   struct LtpDef_i ltp;
   ltp_def_from_ecef_i(&ltp, &gps_nmea.state.ecef_pos);
   ecef_of_ned_vect_i(&gps_nmea.state.ecef_vel, &ltp, &gps_nmea.state.ned_vel);
+
+  /* indicate that a new valid sentence is available */
+  gps_nmea.sentence_available = true;
```
Comment (dewagter) [inline sw/airborne/subsystems/gps/gps_furuno.c]: why is this message sentens_available and not msg_available?

**Comment by dewagter** [comment]
looks great, thanks

File: sw/airborne/subsystems/gps/gps_furuno.c (hunk 1 above)
Comment (flixr) [inline sw/airborne/subsystems/gps/gps_furuno.c]: msg_available means that the char parser got a whole msg/sentence, sentence_available here means that it was actually a valid/supported sentence and gps info was updated...

Maybe nicer to get rid of sentence_available and make parse_msg return true if valid...

**Pull request merged**